src/Movie.elm

```diff
@@ -1,17 +1,27 @@
 module Movie exposing (..)
 
-import Html exposing (Html, div, img, text, a)
-import Html.Attributes exposing (src, href, target)
+import Html exposing (Html, div, img, text, a, button, p, h2, ul, b, li)
+import Html.Attributes exposing (src, href, target, type_, autofocus)
+import Html.Events exposing (onClick)
 import AppCss.Helpers exposing (class, classList)
 import AppCss as Style
 import Time.Date exposing (Date, day, month, year)
 import Set exposing (Set)
 import Genre exposing (Genre)
+import Json.Decode as Decode exposing (string, list)
+import Json.Decode.Pipeline exposing (decode, hardcoded, optional, required)
 
 
 -- Types
 
 
+type MovieSelection
+    = NotSelected
+    | Selected Movie
+    | NothingToSelect
+    | Loaded MovieDetails
```

Comment:
@MattCheely I know you mentioned maybe removing this, but I think it does a good job of representing the four states of a movie selection/focus. This helps when we have a movie selected or focused but if API requests fail (that way it doesn't break pulling a movie from the hat).

Reply:
I wasn't thinking so much that we remove it, but that as far as I can tell, the `NotSelected` and `NothingToSelect` states are identical in the context of the current UI, so we could probably drop the latter.